src/lib/tests/render_with_layout.tsx

```diff
@@ -1,7 +1,7 @@
 import * as renderer from "react-test-renderer"
 
 /** Renders a React Component with specified layout using onLayout callback */
-export const renderWithLayout = (component: any, layout: { width?: number, height?: number }) => {
+export const renderWithLayout = (component: any, layout: { width?: number; height?: number }) => {
```

Comment:
Nah, either should be fine, it’s just a TS interface. Not the prettiest, though, maybe move it out to a named interface?